test/Modules/FreeModules-graded.jl

```diff
@@ -1,326 +1,328 @@
-function _rand_polys_nonzero(R, n)
-  polys = elem_type(R)[]
-  while length(polys) < n
-    f = MPolyBuildCtx(R)
-    for z = 1:3
-      e = rand(0:2, ngens(R))
-      r = base_ring(R)(rand(0:22))
-      push_term!(f, r, e)
-    end
-    g = finish(f)
-    if !iszero(g)
-      push!(polys, g)
+let
```

Comment:
Following what I wrote above, I'd just move the start of the testset up:
```suggestion
@testset "Modules" begin
```

(Plus of course the matching removal of line ~48)